packages/flutter/lib/src/material/app.dart

```diff
@@ -190,6 +193,12 @@ class _MaterialAppState extends State<MaterialApp> {
       return true;
     });
 
-    return result;
+    Widget _iosScroll(Widget scrollWidget) => scrollWidget;
+    Widget _defaultScroll(Widget scrollWidget) => new OverscrollIndicator(child: scrollWidget);
```

Comment:
I'd intended to make these file-level functions, oops. Thanks for pointing that out.